source/grid/manifold.cc

```diff
@@ -577,7 +578,8 @@ FlatManifold<dim, spacedim>::get_new_point(
 {
   Assert(std::abs(std::accumulate(weights.begin(), weights.end(), 0.0) - 1.0) <
            1e-10,
-         ExcMessage("The weights for the individual points should sum to 1!"));
+         ExcMessage("The weights for the individual points should sum up to "
+                    "1!"));
```

Comment:
@bangerth thanks for spotting an accidental change on my end. But considering the proposed text, wouldn't it be more correct with `sum up`?

Reply:
"sum to one" is idiomatic. "sum up to one" does not sound wrong to me, but I would still choose the former.

Reply:
Great, I fixed that.